app/controllers/image_controller.rb

```diff
@@ -805,37 +939,51 @@ def update_licenses_history(images_to_update, old_holder, old_license_id)
 
   ##############################################################################
 
+  # NOTE: These two are more properly account preferences actions
+  # Move to Account::Preferences::ImageVotes#edit and update
+  # Move test from images_controller_test
+  #
   # Bulk update anonymity of user's image votes.
   # Input: params[:commit] - which button user pressed
   # Outputs:
   #   @num_anonymous - number of existing anonymous votes
   #   @num_public    - number of existing puclic votes
   def bulk_vote_anonymity_updater
     if request.method == "POST"
-      submit = params[:commit]
-      if submit == :image_vote_anonymity_make_anonymous.l
-        ImageVote.where(user_id: @user.id).update_all(anonymous: true)
-        flash_notice(:image_vote_anonymity_made_anonymous.t)
-      elsif submit == :image_vote_anonymity_make_public.l
-        ImageVote.where(user_id: @user.id).update_all(anonymous: false)
-        flash_notice(:image_vote_anonymity_made_public.t)
-      else
-        flash_error(
-          :image_vote_anonymity_invalid_submit_button.l(label: submit)
-        )
-      end
-      redirect_to(edit_account_preferences_path)
+      create_anonymity_change
     else
       @num_anonymous = ImageVote.where(user_id: @user.id).
                        where(anonymous: true).
                        pluck(ImageVote[:id].count.as("total"))&.first
       @num_public = ImageVote.where(user_id: @user.id).
                     where(anonymous: false).
                     pluck(ImageVote[:id].count.as("total"))&.first
+    end
+  end
 
+  private
+
+  def create_anonymity_change
```

Comment:
Just a question: I saw this pattern above, too.  "create_some_action".  What does the "create" mean?  Is that code-speak referring to the fact that this is a POST method?

Reply:
Yes. More of a note to self. This is prep for making this one the POST action and the previous action the GET. 

It would probably be `Images::AnonymityController#new` (or maybe `#edit`).

Reply:
Ah, thank!  Totally makes sense.